src/Augmentation/Enums.ts

```diff
@@ -115,10 +115,27 @@ export enum AugmentationName {
   BladeArmorIPU = "BLADE-51b Tesla Armor: IPU Upgrade",
   BladesSimulacrum = "The Blade's Simulacrum",
 
+  // Stanek Augs
   StaneksGift1 = "Stanek's Gift - Genesis",
   StaneksGift2 = "Stanek's Gift - Awakening",
   StaneksGift3 = "Stanek's Gift - Serenity",
 
+  // Charisma augs (needs alphabetization)
+  GoldenTongue = "Golden Tongue Module",
+  Eloquence = "Eloquence Module",
+  Glib = "Glibness Enhancement",
+  Primer = "The Illustrated Primer",
+  Magnetism = "Magnetism Amplifier",
+  SocialDynamo = "Social Dynamics Processor",
+  Wit = "Neural Wit Amplifier",
+
+  // Darknet lab augs
```

Comment:
I'd sort these by enum name too

Reply:
I've updated this to indicate their order is by acquisition, for clarity, but I can re-arrange them if you feel strongly